api/src/main/java/org/openmrs/hl7/impl/HL7ServiceImpl.java

```diff
@@ -539,10 +539,9 @@ public Integer resolveLocationId(PL pl) throws HL7Exception {
 		// HACK: try to treat the first component (which should be "Point of
 		// Care" as an internal openmrs location_id
 		try {
-			Integer locationId = Integer.valueOf(pointOfCare);
-			Location l = Context.getLocationService().getLocation(locationId);
-			if (l != null) {
-				return l.getLocationId();
+			 Integer locationId = Integer.valueOf(pointOfCare);
+	            if (locationId == null) throw new Exception();
+	            return locationId;
 			}
```

Comment:
There seems to be an extra closing bracket there(line 545).

Reply:
Thanks for reviewing my PR . Actually the closing bracket on like 545 is not extra its closing the open bracket on line 534

Reply:
Are you sure? It seems to be standing right between the try and the catch blocks.

Reply:
@HelioStrike  am very sure just click on any bracket it will point you to either opening or closing.

Reply:
```
1. public int somefunc() {
2.     if(1 == 1) {
3.         ...
4.     }
5.     }
6.     System.out.println("I work!!!");
7. }
```
Clicking on the closing bracket on line 5 points to the opening bracket on line 1, but clearly the code is not valid.

That's pretty much what's happening here, except the line number is 546 instead of 5.

Reply:
@tenas97 wasn't wrong either, there's a bracket after the (closing of the)try block, hence the try block had no catch block. Again, deleting the bracket on 546 will compile your program successfully.